client/deposits/overview/instant-deposit-button.js

```diff
@@ -29,34 +66,68 @@ const InstantDepositModal = ( { amount, fee, net, onClose, onSubmit, inProgress
 					currency.formatCurrency( net / 100 )
 				) }
 			</Button>
+			<Button isDefault onClick={ onClose }>
+				{ __( 'Cancel', 'woocommerce-payments' ) }
+			</Button>
 		</Modal>
-	)
-}
+	);
+};
+
+const InstantDepositSubmitNotice = ( { error } ) => {
+	let message = '';
+	// TODO: supply proper error notices.
+	switch ( error.code ) {
+		case 'balance_insufficient':
+			message = __(
+				'Balance is not sufficient for instant deposit.',
+				'woocommerce-payments'
+			);
+			break;
+
+		default:
+			message = __(
+				'There was an error, please try again.',
+				'woocommerce-payments'
+			);
+	}
+
+	return (
+		<Notice status="error" isDismissible={ false }>
+			<p>
+				<strong>{ message }</strong>
```

Comment:
The convention here (at least the unwritten rule I've been following) is that we should do `<p className="something">{ message }</p>` and use the class to style the paragraph in CSS, instead of relying on plain HTML markup for styling.

Reply:
Makes sense, I think this was old habits. I had already created CSS to style the `<p>` based off of the class of the notice. I added the bold font there.